vendor/brave-ios/Ads/BATAdNotification.mm

```diff
@@ -11,6 +11,7 @@ @interface BATAdNotification ()
 @property (nonatomic, copy) NSString *parentUuid;
 @property (nonatomic, copy) NSString *creativeInstanceID;
 @property (nonatomic, copy) NSString *creativeSetID;
+@property (nonatomic, copy) NSString *campaignID;
```

Comment:
I don't see anything in particular (aside from the change to the ads_service_impl at the top), but is this new property supposed to be used in the client in a specific way?

Reply:
Also missing the read-only property definition for this in `BATAdsNotification.h`

Reply:
This new property is not used in the client as of yet

Reply:
Fixed